nullaway/src/main/java/com/uber/nullaway/AbstractConfig.java

```diff
@@ -86,7 +86,15 @@ public boolean fromAnnotatedPackage(String className) {
 
   @Override
   public boolean isExcludedClass(String className) {
-    return sourceClassesToExclude != null && sourceClassesToExclude.contains(className);
+    if (sourceClassesToExclude == null) {
+      return false;
+    }
+    for (String classPrefix : sourceClassesToExclude) {
+      if (className.startsWith(classPrefix)) {
+        return true;
+      }
+    }
```

Comment:
Obviously there are faster implementations than a set of string prefixes and a for loop, but it doesn't seem worth optimizing this yet. Internally, this set currently has a very small number of entries.